packages/webservice/app/src/main/resources/logback-access-spring.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <springProperty scope="context" name="logDir" source="logging.path"/>
+
+    <appender name="ACCESSFILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>${logDir}/access.log</file>
+        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+            <fileNamePattern>${logDir}/archived/access_%d{yyyy-MM-dd}.log</fileNamePattern>
+            <maxHistory>30</maxHistory>
+            <totalSizeCap>100MB</totalSizeCap>
+        </rollingPolicy>
+        <encoder>
+            <pattern>%t{yyyy-MM-dd:HH:mm:ss Z} %remoteIP %user %requestURL %statusCode %bytesSent %elapsedTime %header{X-B3-TraceId} %requestContent %responseContent</pattern>
```

Comment:
This logs the response bodies - is that OK for security?

Reply:
Most likely users will need to configure this to the way they want. I pulled this from the spring-elide example. Do you think this is a bad default? Could be helpful for debugging when starting out.

Reply:
I don't think we want to put presto data in our logs.

Reply:
This is the example repo